daprdocs/content/en/reference/components-reference/supported-state-stores/setup-postgresql-v1.md

```diff
@@ -100,8 +110,8 @@ Authenticating with Microsoft Entra ID is supported with Azure Database for Post
 
      > This example does not describe a production configuration because it sets the password in plain text and the user name is left as the PostgreSQL default of "postgres".
 
-2. Create a database for state data.
-Either the default "postgres" database can be used, or create a new database for storing state data.
+2. Create a database for state data.  
```

Comment:
```suggestion
1. Create a database for state data.  
```

Reply:
There's a #1 above

Reply:
We can have it either way, honestly - in Markdown, only the first number in a numbered list matters. I simply changed this one bc I typically use:

```
1. text
1. text
1. text
```

bc it will render as:

```
1. text
2. text
3. text
```

and I don't have to think about it haha...

source: https://www.markdownguide.org/basic-syntax/#ordered-lists